docs/advanced-features/workflows.md

```diff
@@ -299,8 +301,10 @@ objects known as *package entries*. Each package entry defines a logical key
 ```yaml
 workflows:
   myworkflow-1:
+    name: My workflow №1
     entries_schema: must-contain-readme
   myworkflow-2:
+    name: My workflow №2
```

Comment:
in English using `#` is more common than `№`

Reply:
Yes, but wouldn't it be treated as comment in YAML?

Reply:
I think you can quote the string:
```suggestion
    name: 'My workflow #2'
```